osu.Server.Queues.ScorePump/Performance/Scores/UpdateScoresForUsers.cs

```diff
@@ -7,10 +7,10 @@
 using JetBrains.Annotations;
 using McMaster.Extensions.CommandLineUtils;
 
-namespace osu.Server.Queues.ScorePump.Performance.Values
+namespace osu.Server.Queues.ScorePump.Performance.Scores
 {
-    [Command("users", Description = "Computes pp of specific users.")]
-    public class UpdateValuesUsersCommand : PerformanceCommand
+    [Command("userlist", Description = "Computes pp of all scores of specific users.")]
```

Comment:
Just confirming, you intended `userlist` instead of `users`?

Reply:
Yeah... because it gets confusing otherwise like `dotnet run batch scores users` and you also have `dotnet run batch user-totals`

Would be resolved if the filter commands are combined as I touch on in OP. Can look at doing that if you agree with the direction.

Reply:
I'm like 60% in agreement only because I can foresee issues with what if multiple options are given. But at the end of the day you're the one going to be using these commands, so...

Reply:
Will leave them for now and see how they feel.